src/openrct2/ride/Track.h

```diff
@@ -52,6 +52,13 @@ struct rct_track_coordinates
     int16_t y;             // 0x08
 };
 
+enum
+{
+    RCT_PREVIEW_TRACK_FLAG_1 = (1 << 0),
```

Comment:
Is this coincident through the code base i thought we normally started at 0

Reply:
Wasn't sure myself. I'll change it to 0.